samtranslator/utils/utils.py

```diff
@@ -49,3 +49,25 @@ def dict_deep_get(d: Any, path: str) -> Optional[Any]:
         relative_path = (relative_path + f".{_path_nodes[0]}").lstrip(".")
         _path_nodes = _path_nodes[1:]
     return d
+
+
+def dict_deep_set(d: Any, path: str, value: Any) -> None:
+    """
+    Get the value deep in the dict.
```

Comment:
```suggestion
    Set the value deep in the dict.
```

Also I like this name better, makes much more sense. 👍 Wasn't aware of the merge/override thing that `path` dealt with.